### DISCUSSION TITLE
Sublime Text syntax for FPP

### DISCUSSION THREAD

**Discussion by walkingbeard** [nasa/fprime#2219] [Q&A]
Has anyone already made a Sublime Text syntax for FPP that they would be willing to share? I have only ever modified my C syntax for my own types, rather than building one from scratch, and it would be a waste of time for me if someone has already done it.

**Comment by SterlingPeet** [comment]
I am also interested in this.

**Reply by LeStarch** [reply]
I am not aware of any, but I'll keep an ear to the ground.  Currently, I know of support for:
1. VsCode
2. VIM
